### ISSUE TITLE
Invalid data cell.

### ISSUE THREAD

**Issue by rvhonorato** [samirelanduk/molstar-react#5]
When trying to run even the example in the documentation I'm getting this error:

```
Uncaught (in promise) Error: Invalid data cell.
    parseTrajectory index.js:108809
    _callee2$ index.js:118978
    Async Babel
    node_modules bundle.js:213649
    node_modules bundle.js:213691
    asyncGeneratorStep index.js:8
    _next index.js:30
    promise callback*asyncGeneratorStep index.js:18
    _next index.js:30
    _asyncToGenerator index.js:37
    _asyncToGenerator index.js:26
    loadStructure index.js:118994
    _callee$ index.js:118868
    Async Babel
    node_modules bundle.js:213649
    node_modules bundle.js:213691
    asyncGeneratorStep index.js:8
    _next index.js:30
    promise callback*asyncGeneratorStep index.js:18
    _next index.js:30
    _asyncToGenerator index.js:37
    _asyncToGenerator index.js:26
    Molstar index.js:118876
    React 13
    workLoop scheduler.development.js:266
    flushWork scheduler.development.js:239
    performWorkUntilDeadline scheduler.development.js:533
    js scheduler.development.js:571
    js scheduler.development.js:633
    factory react refresh:6
    Webpack 24
index.js:108809
    asyncGeneratorStep index.js:11
    _throw index.js:34
    (Async: promise callback)
    asyncGeneratorStep index.js:18
    _next index.js:30
    (Async: promise callback)
    asyncGeneratorStep index.js:18
    _next index.js:30
    _asyncToGenerator index.js:37
    _asyncToGenerator index.js:26
    Molstar index.js:118876
    React 8
    performSyncWorkOnRoot self-hosted:1406
    React 5
    performConcurrentWorkOnRoot self-hosted:1406
    workLoop scheduler.development.js:266
    flushWork scheduler.development.js:239
    performWorkUntilDeadline scheduler.development.js:533
    (Async: EventHandlerNonNull)
    js scheduler.development.js:571
    js scheduler.development.js:633
    factory react refresh:6
    Webpack 24
```

Sandbox with error: https://codesandbox.io/s/euyt1p

Any suggestion on how to bypass this?

**Comment by Finsam** [comment]
Same here, is this related to using React v18.x.x?

**Comment by samirelanduk** [comment]
Fixed by @SyLi9527 in #8 - apologies for getting back on this sooner.
